test_connections.py

```diff
@@ -0,0 +1,12 @@
+import urllib.request
+import pytest
+
+SITE_LOCATION = "http://localhost:5000"
+
+
+def connection():
```

Comment:
why not call this function test_connction and simple do an assert here? PyTest provides you with ways to assert that something does not throw an exception without having to write the exception handling code on your own.